azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CURLTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.b2c;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class AADB2CURLTest {
+
+    @Test
+    public void testGetAuthorizationUrl() {
+        final String expect = "https://fake-tenant.b2clogin.com/fake-tenant.onmicrosoft.com/oauth2/v2.0/authorize";
```

Comment:
Do we need to add some code about how this url is generated from test constants? As if the test case is failed, others may not understand why it is failed and how to fix it.

Reply:
make sense. But the code for generate the test constant may be the same as implement code. How about add some comments here and link to implement code ?